### DISCUSSION TITLE
Your models in app(s): (... )  have changes that are not yet reflected in a migration, and so won't be applied.

### DISCUSSION THREAD

**Discussion by RogueRopeMaster** [pretix/pretix#2189] [Q&A] [answered]
pretix@pretix:~/venv$ python -m pretix migrate
Operations to perform:
  Apply all migrations: auth, badges, banktransfer, contenttypes, oauth2_provider, otp_static, otp_totp, paypal, pretixapi, pretixbase, pretixdroid, pretixhelpers, pretixmultidomain, sendmail, sessions, stripe, ticketoutputpdf

Running migrations:

  No migrations to apply.

  **Your models in app(s): 'badges', 'banktransfer', 'pretixapi', 'pretixbase', 'pretixmultidomain', 'sendmail', 'ticketoutputpdf' have changes that are not yet reflected in a migration, and so won't be applied.**

After performing upgrade, installing some plugins and changing the domain in config and nginx. 
Is this normal? Something to worry about? Did I do something wrong?

**Comment by raphaelm** [comment, marked as answer]
This is normal and safe to ignore. We unfortunately cannot easily fix it.

**Reply by RogueRopeMaster** [reply]
Ah, ok, thanks. The red font made it a bit scary. 
Thanks for your efforts!
